expense-notifications-functions/lib/index.js

```
@@ -0,0 +1,404 @@
+}
+// Helper function to format currency
+function formatCurrency(amount) {
+    const formatter = new Intl.NumberFormat('en-IN', { style: 'currency', currency: 'INR' });
+    return formatter.format(amount);
+}
+// Function to calculate the owed or payback amount for a member
+function calculateOwedAmount(expenseData, memberId) {
+    const splitTo = expenseData.split_to;
+    const splitAmount = getTotalSplitAmountOf(expenseData, memberId);
+    const paidAmount = expenseData.paid_by[memberId] || 0;
+    if (expenseData.paid_by.hasOwnProperty(memberId)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)